pub_topic: "/zed_objdetect/left/image_rect_color"
framerate: 30
show_video: true
empty_video: false # Instead of displaying a video, send an empty 1080p image, mainly useful for triggering trtdetect.py to start initializing

#Edit me!
filename: "testvid.mp4"
